portal/settings.py

```diff
@@ -92,8 +92,8 @@
         'default': {
             'ENGINE': 'django.db.backends.postgresql_psycopg2',
             'NAME': 'dscatc',
-            'USER': 'name',
-            'PASSWORD': '',
+            'USER': 'harsh',
+            'PASSWORD': '1234',
```

Comment:
You can't change settings as per your local machine settings, this might work for you but break for other people

Reply:
okay cool...i'll change them again to default